integration/testrunner/common.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Lumberjack authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testrunner
+
+import (
+	"net/mail"
+	"testing"
+
+	"cloud.google.com/go/logging/apiv2/loggingpb"
+)
+
+func diffLogEntry(tb testing.TB, logEntry *loggingpb.LogEntry, requireJustification bool) {
```

Comment:
Do you plan to make the logs consistent-ish in integ test for easier comparison? If so, add a TODO here.

Reply:
Yes. It needs further efforts. In addition, I found java side doesn't set timestamp in the logentry. I will create separate issues and add TODOs here.

Reply:
Thanks for catching that. It's weird because in my understanding cloud logging will set the timestamp if we don't set it.

Reply:
done